utilities_common/cli.py

```diff
@@ -309,6 +415,65 @@ def is_pc_router_interface(config_db, pc):
 
     return False
 
+
+def get_vlan_id(vlan):
+    vlan_prefix, vid = vlan.split('Vlan')
+    return vid
+
+def get_interface_name_for_display(db ,interface):
+    interface_naming_mode = get_interface_naming_mode()
+    iface_alias_converter = InterfaceAliasConverter(db)
+    if interface_naming_mode == "alias" and interface:
+        return iface_alias_converter.name_to_alias(interface)
```

Comment:
>return [](http://example.com/codeflow?start=8&length=6)

Could you add a testcase to cover this line?

Reply:
This is your new code :)
If you mean you copied code from elsewhere, it is still your responsibility to test/cover it.

Reply:
Okay will add more test cases as well